modules/controlpanel.cpp

```diff
@@ -1130,7 +1132,7 @@ class CAdminMod : public CModule {
         CString sPass = sLine.Token(5);
 
         if (sServer.empty()) {
-            PutModule("Usage: DelServer <username> <network> <server>");
+            PutModule("Usage: DelServer <username> <network> <server> [port] [password]");
```

Comment:
For what it's worth, I use docs.znc.in for my development, which does not point out this quirk.